packages/rpm/polymc.spec

```diff
@@ -19,37 +80,60 @@ Requires: pkgconfig(Qt5)
 Requires: pkgconfig(zlib)
 
 %description
-A custom launcher for Minecraft that allows you to easily manage multiple installations of Minecraft at once
+PolyMC is a free, open source launcher for Minecraft. It allows you to have
+multiple, separate instances of Minecraft (each with their own mods, texture
+packs, saves, etc) and helps you manage them and their associated options with
+a simple interface.
+
 
 %prep
-mkdir -p %{_builddir}/%{name}
-cp -r %{_origdir}/../../* %{_builddir}/%{name}
+%autosetup -p1 -n PolyMC-%{version}
+
+tar -xvf %{SOURCE1} -C libraries
+tar -xvf %{SOURCE2} -C libraries
+rmdir libraries/libnbtplusplus libraries/quazip
+mv -f libraries/quazip-%{quazip_commit} libraries/quazip
+mv -f libraries/libnbtplusplus-%{libnbtplusplus_commit} libraries/libnbtplusplus
+
 
 %build
-cd %{_builddir}/%{name}
-cmake \
-  -DLauncher_LAYOUT=lin-system \
-  -DCMAKE_INSTALL_PREFIX=/usr \
-  -DLauncher_LIBRARY_DEST_DIR=%{_lib} \
-  .
+%cmake \
+    -DCMAKE_BUILD_TYPE:STRING="RelWithDebInfo" \
+    -DLauncher_LAYOUT:STRING="lin-system" \
+    -DLauncher_LIBRARY_DEST_DIR:STRING="%{_libdir}/%{name}" \
```

Comment:
the shared objects build by PolyMC are very PolyMC specific and cannot be relied on by anything else so they are moved into their own directory